### PR TITLE
README improvements towards 0.1

### PR DESCRIPTION
**Pull request by avital**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ Changes will come to the API, but we'll use deprecation warnings when we can, an
 keep track of them our [Changelog](CHANGELOG.md).
 
 A growing community of researchers at Google are happily using
-Flax daily for their research, and now we'd like to extend that support to the
-open source community. GitHub issues are encouraged for open converation, but
-in case you need to reach us directly, we're at flax-dev@google.com.
+for their daily research and contributing to it, and now we're
```
Comment (levskaya) [inline README.md]: happily using ^Flax for

Comment (levskaya) [inline README.md]: ...happily using ^Flax for their...

File: README.md [hunk 2]
```diff
@@ -5,11 +5,12 @@ Changes will come to the API, but we'll use deprecation warnings when we can, an
...
+expanding that to the open source community.
 
-## Quickstart
+[GitHub issues](http://github.com/google/flax/issues) are encouraged for open converation, but
```
Comment (levskaya) [inline README.md]: "converSation"

**Comment by avital** [comment]
Thanks! Made those fixes and will merge now.

**Pull request merged**